pkg/client/userd/trafficmgr/mocks/clusterbuilder_mock.go

```diff
@@ -0,0 +1,52 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: github.com/telepresenceio/telepresence/v2/pkg/client/userd/trafficmgr (interfaces: ClusterBuilder)
+
+// Package mock_trafficmgr is a generated GoMock package.
+package mock_trafficmgr
```

Comment:
Won't all this mock-code end up in the runtime binary? In other words,  shouldn't it all reside in xxx_test.go files?